rollup.config.js

```diff
@@ -38,7 +38,12 @@ export default () =>
             exclude: [`${pkgPath}/src/**/*.test.tsx`, `${pkgPath}/src/**/*.test.ts`, `${pkgPath}/src/**/*.stories.tsx`],
           },
         }),
-        postcss(),
+        postcss({
+          extract: false,
+          modules: {
+            generateScopedName: '[hash:base64]',
```

Comment:
Should we limit the length of the hash here? Not sure what the default length would be

Reply:
It ends up like this: 

![image](https://user-images.githubusercontent.com/11799835/155418960-d940c3b1-1199-4971-9211-45dabaa23616.png)

So that's 20 characters by my count. Yeah, that does seem excessive to me. Do you have a suggestion @chrishavekost? 

I followed [this recommendation from the docs for CSS Loader](https://webpack.js.org/loaders/css-loader/#localidentname): 

> Recommendations:
> 
> - use '[path][name]__[local]' for development
> - use '[hash:base64]' for production

@jordan-a-young and I talked about this a bit. We both liked the idea of leaving the file and original class name in place, with a much smaller hash. So in this case it would be something like `FavoriteHeart__icons__BFYtN`.

I like this because it will make debugging easier for our devs. Plus it feels very familiar to me from my use of BEM. Given that these components will be in the development environments of our devs as they work, obscuring the class names with nothing but a hash seems less than ideal. Unless we publish production and development versions of our components (like React does), it will be difficult for devs to identify in DevTools which DOM elements are coming from our library components.

The only downside to this right now is that the convention for CSS Modules calls for filenames like `FavoriteHeart.module.scss`. This means that `generateScopedName: '[name]__[local]__[hash:base64:5]'` would produce a class name like `FavoriteHeart-module__icons__BFYtN`. This extra '-module' bit seems pretty ugly to me for production code. Create React App goes to some lengths to remove this in their [react-scripts webpack config](https://github.com/facebook/create-react-app/blob/5614c87bfbaae0ce52ac15aedd2cd0f91ffd420d/packages/react-scripts/config/webpack.config.js#L25) with [some fancy footwork](https://github.com/facebook/create-react-app/blob/5614c87bfbaae0ce52ac15aedd2cd0f91ffd420d/packages/react-dev-utils/getCSSModuleLocalIdent.js#L38-L39) in their react-dev-utils package.

In summary, I have given some thought to this, but the best solution is probably out of scope for my ticket. If you'd like to make a suggestion for the number of characters for the hash, though, I'm all ears.

Reply:
I changed it to `[hash:base64:12]`